integration/ui/satellite/signup_personal_user_test.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2021 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellite
+
+import (
+	"testing"
+	"time"
+
+	"github.com/go-rod/rod"
+	"github.com/go-rod/rod/lib/input"
+	"github.com/stretchr/testify/require"
+
+	"storj.io/common/testcontext"
+	"storj.io/storj/integration/ui/uitest"
+	"storj.io/storj/private/testplanet"
+)
+
+func TestPersonalUserCanSignUp(t *testing.T) {
+	uitest.Run(t, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet, browser *rod.Browser) {
+		signupPageURL := planet.Satellites[0].ConsoleURL() + "/signup"
+		fullName := "John Doe"
+		emailAddress := "test@email.com"
+		password := "qazwsx123"
+		page := browser.Timeout(10 * time.Second).MustPage(signupPageURL)
+		page.MustSetViewport(1350, 600, 1, false)
+		// First time User signup
+		page.MustElement(".headerless-input").MustInput(fullName)
+		page.MustElementX("//body/div[@id='app']/div[1]/div[2]/div[1]/div[4]/div[1]/input[1]").MustInput(emailAddress)
```

Comment:
This kind of search for appropriate fields will be very fragile. You should never put in the full path of an html element.

Let frontend team put in appropriate tags/ids/other-info if it's difficult to find the correct thing.